scss/components/_dropdown-menu.scss

```diff
@@ -207,7 +207,7 @@ $dropdownmenu-border-width: nth($dropdownmenu-border, 1);
       }
     }
 
-    > li {
+    > li:not('.column') {
       width: 100%;
```

Comment:
- Why do we automatically apply this property on all `<li>` in the first place, instead of using a class ?
- Why do we declare again every menu-related properties inside a `dropdown-menu` component, instead of simplifying `dropdown` and `menu` ?

Also, like [here](https://github.com/zurb/foundation-sites/pull/9098#pullrequestreview-3417021), we can't assume the behavior the user expect by the classes he uses. Filtering `.column` fix this particular case and nothing more, can make weird behavior, and lead to an unmaintainable spaghetti code.